truffle.js

```diff
@@ -5,6 +5,13 @@ module.exports = {
             port: 8545,
             network_id: "*" // Match any network id
         },
+        coverage: {
+            host: "localhost",
+            network_id: "*",
+            port: 8555,
+            gas: 0xfffffffffff,
+            gasPrice: 0x01
```

Comment:
This is useful for testing, however there is still no way to deploy Futarchy Oracles to live testnets or mainnet unfortunately.